features-connection.js

```diff
@@ -186,6 +192,30 @@ module.exports = {
         return lifeTime > 0 ? lifeTime : null;
     },
 
+    sendingDuration: function(client, peerConnectionLog) {
+        let sendingDuration = 0;
+        let prevTime = timestamp(peerConnectionLog[0]);
+        let prevSending = false;
+        peerConnectionLog.forEach(log => {
+            if (log.type !== 'setLocalDescription') {
+                return;
+            }
+            const sections = SDPUtils.getMediaSections((log.value && log.value.value && log.value.value.sdp) || '');
+            const direction =  SDPUtils.getDirection(sections[0] || '');
```

Comment:
this will blow up if sections is not an array. Don't try half-baked data validation :-)

Reply:
.value.value.sdp can't be right?